Source/santad/SNTPolicyProcessor.mm

```diff
@@ -128,8 +128,8 @@ - (BOOL)decision:(SNTCachedDecision *)cd
                      forRule:(SNTRule *)rule
          withTransitiveRules:(BOOL)enableTransitiveRules
     andCELActivationCallback:(ActivationCallbackBlock)activationCallback {
-  if (rule.state == SNTRuleStateCEL) {
-    auto activation = activationCallback ? activationCallback() : nullptr;
+  if (rule.state == SNTRuleStateCEL && activationCallback) {
```

Comment:
Just confirming - the plan is to better handle the `santactl fileinfo` path in a future PR, yeah?